src/sprite_battler.h

```diff
@@ -42,7 +43,8 @@ class Sprite_Battler : public Sprite {
 		AnimationState_WalkingLeft,
 		AnimationState_WalkingRight,
 		AnimationState_Victory,
-		AnimationState_Item
+		AnimationState_Item,
+		AnimationState_SelfDestruct
```

Comment:
Adding anything after Item feels misleading btw because this enum maps exactly to "Animation 2 -> Animation Type" in the RPG Maker 2003 editor and it is possible to have up to 32 different poses (for Items and Skills).

The 2k3 battle system will never read this SelfDestruct enum for actors but this shows that Sormat didn't notice where the enum values come from.

And yes, I know that I heavily abused this state stuff to inject "animations" for enemies 😜 (but at least they mapped to existing 2k3 anim stuff)

Reply:
> Shouldn't such an enum be in liblcf?

yes

That's not 2k business, this is monster business. For no logical reason they decided to keep the monsters static and only the party battlers can be animated. (there is "flying" but this only moves the static sprite up and down)

Reply:
I moved SelfDestruct = 255, so it's out of the way of items.